src/views/ReviewConfirm.vue

```diff
@@ -6,6 +6,16 @@
       </header>
       <Summary />
     </section>
+    <section class="mt-10">
+      <header>
+        <h2>Incorporation Date Time</h2>
```

Comment:
Header is incorrect and you're missing the instructions, per https://preview.uxpin.com/6bd383a032304acc6ef186307cf186993fb02a39#/pages/127750603/simulate/no-panels .

Reply:
Or https://preview.uxpin.com/d7266b8b6fb8bd80acc0ff38ff5c946434492f5e#/pages/122406228/simulate/no-panels .

Reply:
Definitely, thanks for the reminder. Scott mentioned to add this in!

Reply:
Added!

Reply:
H2 text is still incorrect 😬